src/components/Form.tsx

```diff
@@ -51,7 +78,18 @@ const Form = ({ entryDate, isAuthorized, data, updateEntryDate, selections, setS
     return () => setError(null);
   }, [isAuthorized, entryDate, data, router, setSelections]);
 
-  const postForm = async () => {
+  useEffect(() => {
+    const handleScroll = () => {
+      setScrolling(window.scrollY > 0);
```

Comment:
This "window.scrollY > 0" logic (and conditional BG color) shows up in two places. If only just to help reduce the scope of responsibility of this Form component, wdyt about abstracting this into a separate component? Perhaps a ScrollYContext, and/or OpaqueOnScrollY component (with conditional BG)...